applications/StructuralMechanicsApplication/python_scripts/distribute_force_on_surface_process.py

```diff
@@ -0,0 +1,64 @@
+import KratosMultiphysics as KM
+import KratosMultiphysics.StructuralMechanicsApplication as KSM
+
+
+def Factory(settings, Model):
+    if not isinstance(settings, KM.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return DistributeForceOnSurfaceProcess(Model, settings["Parameters"])
+
+
+class DistributeForceOnSurfaceProcess(KM.Process):
+    """This process distributes a force on surface load conditions belonging to a submodelpart.
+    The force is distributed according to the surface area.
+    """
+    def __init__(self, model, settings):
+        """ The default constructor of the class
+
+        Arguments:
+        model -- the container of the different model parts.
+        settings -- Kratos parameters containing process settings.
+        """
+        KM.Process.__init__(self)
+
+        # The value can be a double or a string (function)
+        default_settings = KM.Parameters("""
+        {
+            "help"            : "This process distributes a force on surface load conditions belonging to a submodelpart. The force is distributed according to the surface area.",
+            "model_part_name" : "please_specify_model_part_name",
+            "interval"        : [0.0, 1e30],
+            "force"           : [1.0, 0.0, 0.0]
+        }
+        """)
+
+        # Assign this here since it will change the "interval" prior to validation
+        self.interval = KM.IntervalUtility(settings)
+
+        settings.ValidateAndAssignDefaults(default_settings)
+
+        self.variable_utils = KM.VariableUtils()
+
+        self.model_part = model.GetModelPart(settings["model_part_name"].GetString())
+        self.force = [
+            settings["force"][0].GetDouble(),
+            settings["force"][1].GetDouble(),
+            settings["force"][2].GetDouble()
+        ]
```

Comment:
```suggestion
        self.force = settings["force"].GetVector()
```

plus a check to make sure the size is 3